samples/snippets/src/main/java/com/example/analyticsadmin/QuickstartSample.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.analytics;
```

Comment:
this could also be just `package analytics`

Reply:
Technically yes, but that's a big violation of Java conventions

Reply:
I would prefer to leave the package name as is. This is consistent with almost every sample I was able to find in Cloud, and looks more intuitive.